client/my-sites/comments/comment-list/index.jsx

```diff
@@ -475,8 +475,8 @@ const mapStateToProps = ( state, { siteId, status } ) => {
 	const comments = map( getSiteCommentsTree( state, siteId, status ), 'commentId' );
 	const isLoading = ! isCommentsTreeInitialized( state, siteId, status );
 	return {
+		isCommentsTreeSupported: ! isJetpackSite( state, siteId ) || isJetpackMinimumVersion( state, siteId, '5.3' ),
```

Comment:
Let's bump this to `5.5`, which will be the next release, with the [new speedier response shape](https://github.com/Automattic/jetpack/pull/7933).